gui/elm/Address.elm

```diff
@@ -56,6 +86,9 @@ update msg model =
         FillAddress address ->
             ( { model | address = address, error = "", busy = False }, Cmd.none )
 
+        CorrectAddress ->
+            ( { model | address = correctAddress model.address }, Cmd.none )
```

Comment:
I expected it to be done in *core/app* since I believe there is already some validation in there. Although I definitely prefer the elm implementation :trollface: